webapp/src/Entity/BlogPost.php

```diff
@@ -0,0 +1,179 @@
+<?php declare(strict_types=1);
+
+namespace App\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+use JMS\Serializer\Annotation as Serializer;
+use OpenApi\Annotations as OA;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * Public blog posts sent by the jury
+ *
+ * @ORM\Entity()
+ * @ORM\Table(
+ *     name="blog_post",
+ *     options={"collation"="utf8mb4_unicode_ci", "charset"="utf8mb4", "comment"="Public blog posts sent by the jury"},
+ *     indexes={
+ *         @ORM\Index(name="slug", columns={"slug"}),
+ *     },
+ *     uniqueConstraints={
+ *         @ORM\UniqueConstraint(name="slug", columns={"slug"})
+ *     })
+ * @UniqueEntity("slug")
+ */
+class BlogPost extends BaseApiEntity
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     * @ORM\Column(type="integer", length=4, name="blogpostid",
+     *     options={"comment"="Blog post ID","unsigned"=true},
+     *     nullable=false)
+     * @Serializer\SerializedName("id")
+     * @Serializer\Type("string")
+     */
+    protected int $blogpostid;
+
+    /**
+     * @ORM\Column(type="string", name="slug", length=255,
+     *     options={"comment"="Unique slug"},
+     *     nullable=false)
+     * @Serializer\Exclude()
+     */
+    private string $slug;
+
+    /**
+     * @var double|string
+     * @ORM\Column(type="decimal", precision=32, scale=9, name="publishtime", options={"comment"="Time sent", "unsigned"=true}, nullable=false)
+     * @Serializer\Exclude()
+     */
+    private $publishtime;
+
+    /**
+     * @ORM\Column(type="string", name="author", length=255,
+     *     options={"comment"="Name of the post author"},
+     *     nullable=true)
+     * @Serializer\Exclude()
+     */
+    private ?string $author;
+
+    /**
+     * @ORM\Column(type="string", name="title", length=511,
+     *     options={"comment"="Blog post title"},
+     *     nullable=false)
+     * @Serializer\Exclude()
+     */
+    private string $title;
+
+    /**
+     * @ORM\Column(type="text", length=4294967295, name="subtitle",
+     *     options={"comment"="Blog post subtitle"},
+     *     nullable=false)
+     * @Serializer\SerializedName("subtitle")
+     */
+    private string $subtitle;
+
+    /**
+     * @ORM\Column(type="string", name="thumbnail_file_name", length=255,
+     *     options={"comment"="Thumbnail file name"},
+     *     nullable=false)
+     * @Serializer\Exclude()
+     */
+    private ?string $thumbnail_file_name;
```

Comment:
php standard je spis camelCase

Reply:
v [Clarification](https://github.com/delta-cs/seminar-domjudge/blob/d6c7c7dd2434730e10bb33ee643b3216be869615/webapp/src/Entity/Clarification.php#L68) maji snake

ale rikal jsem si to stejne, nevim proc to tak maji. jestli kvuli shodnosti s DB? to bych cekal, ze budou zajistovat ty ORM notace.

Reply:
ok, tak drzme klidne konvenci. Je mozne, ze to souvisi nejak s mapovanim